app/k9mail/src/main/java/com/fsck/k9/widget/unread/UnreadWidgetConfigurationFragment.kt

```diff
@@ -32,6 +32,27 @@ class UnreadWidgetConfigurationFragment : PreferenceFragmentCompat() {
     private var selectedAccountUuid: String? = null
     private var selectedFolderId: Long? = null
 
+    override fun onSaveInstanceState(outState: Bundle) {
```

Comment:
In the K-9 Mail code base typically one of the `onCreate*()` methods is the first one in a Fragment/Activity. Please move this method below `onCreatePreferencesFix()`.